onnxscript/type_annotation.py

```diff
@@ -13,24 +13,36 @@
 
 from onnxscript.onnx_types import TensorType
 
+# TypeAnnotationValue represents the (value of) valid type-annotations recognized
+# by ONNX Script. TODO: Flesh out a formal definition. Currently, it supports
+# * float, int, str (primitive attribute types)
+# * Sequence[float], Sequence[int], Sequence[str] (attribute types)
+# * Tensor types
+# * Sequence[Tensor] types
+# * Union of above 2
+# * TypeVars with above bounds
+# * Above types with annotation attached
+TypeAnnotationValue = typing.Any
+
+# Map from python type to corresponding ONNX AttributeProto type
 _PYTYPE_TO_ATTRTYPE_MAP = {
     float: onnx.AttributeProto.FLOAT,
     int: onnx.AttributeProto.INT,
     str: onnx.AttributeProto.STRING,
-    bool: onnx.AttributeProto.INT,
```

Comment:
👀

Reply:
(Weird UI ... my comment disappeared.) I had to add bool as many aten examples use it, and I was running into issues there.